### PR TITLE
README clarifications

### PR DESCRIPTION
**Pull request by damdoy**
Add missing infos in the readme: flash jumpers, riscv link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +29,34 @@ Most of the examples use the 12MHz external clock from the breakout board. The P
...
-Each example can be compiled with a `make` which will create the bitstream using the icestorm opensource tools, once the breakout board is plugged, `make prog` will program the fpga using the sram, `make prog_flash` will program the flash of the fpga.
+Each example can be compiled with a `make` which will create the
+bitstream using the icestorm opensource tools, once the breakout board
+is plugged.
+
+`make prog` will program the fpga using the internal sram based condiguration memory in
+the Ice40 Ultraplus device. Since this memory is volatile, it will get
+erased every time the board is powered down.
+
+`make prog_flash` will will write the FPGA configuration to the separate
```
Comment (damdoy) [inline README.md:40]: Duplication of "will"

File: README.md [hunk 2]
```diff
@@ -25,12 +29,34 @@ Most of the examples use the 12MHz external clock from the breakout board. The P
...
 ### Versions used
 
-Yosys 0.9  
-arachne-pnr 0.1+325+0  
+Yosys 0.9
+arachne-pnr 0.1+325+0
 nextpnr-ice40 (git sha1 c365dd1)
-gcc version 5.4.0  
+gcc version 5.4.0
 Built on Linux Mint 18.2
```
Comment (damdoy) [inline README.md:62]: This will display everything on the same line in markdown, it should maybe be a list

Comment (secworks) [reply README.md:62]: Seems Emacs removed trailing whitespace. And yes it could be a list.
I'll fix.

**Comment by secworks** [comment]
Ok, commited a fix for both issues.
https://github.com/secworks/ice40_ultraplus_examples/commit/859ceb9e2072a983f98c68d659d9f9ad82752d7c

**Pull request merged**